src/Illuminate/Validation/Rules/ArrayRule.php

```diff
@@ -28,25 +30,136 @@ public function __construct($keys = null)
             $keys = $keys->toArray();
         }
 
-        $this->keys = is_array($keys) ? $keys : func_get_args();
+        $keys = is_array($keys) ? $keys : func_get_args();
+
+        if (empty($keys)) {
```

Comment:
You could do this without a function call:
```suggestion
        if ($keys === []) {
```

Reply:
yes you are right i update it